packages/web3-eth-contract/test/integration/contract_erc721.test.ts

```diff
@@ -136,8 +133,8 @@ describe('contract', () => {
 							),
 							pkAccount.privateKey,
 						);
-						const logs = await contractDeployed.getPastEvents('Transfer');
-						const tokenId = (logs[0] as EventLog)?.returnValues?.tokenId as string;
+						// eslint-disable-next-line
```

Comment:
The linter should be complaining about this, but you should provide the rule you're intending to disable

Reply:
done